packages/core-app-api/src/lib/AuthConnector/types.ts

```diff
@@ -28,3 +28,13 @@ export type AuthConnector<AuthSession> = {
   refreshSession(scopes?: Set<string>): Promise<AuthSession>;
   removeSession(): Promise<void>;
 };
+
+/**
+ * Popup options for login popup
+ * @public
+ */
+export type PopupOptions = {
+  fullscreen?: boolean;
```

Comment:
Setting width or height to 0 will also cause it to go fullscreen, perhaps that's enough? Although thinking a bit more about it I feel that these should perhaps be groped a bit. What do you think about the following?

```ts
interface PopupOptions {
  size?: { width: number; height: number } | { fullscreen: true }
}
```

That way you avoid a situation where only width or height is set, or all 3 are set.